internal/diagnostics/diagnostics.go

```diff
@@ -1,18 +1,5 @@
 package diagnostics
 
-type ReleaseVersion string
-
 var Version string
 var Commit string
 var BuildDate string
-
-const (
```

Comment:
We still need some way to have a "running flavour" to allow the app to branch in and out depending on the machine / conditions it is running. We removed this in the case of the storage devices on this PR, but this is and will be necessary for other configuration aspects. How should we deal with this?

- Pass a flag at build time (aka `FLAVOUR`)
- Do it dynamically at run time based on some environment property (file, env var, etc), and then deal with setting this via the installers for specific platforms.

Wdyt?